roster/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+from uuid import uuid4
+from django.contrib.auth.models import User
+
+class Site(models.Model):
```

Comment:
I like "site", but this is "Org" in OneRoster data model, so we should consider whether that's better.